releases/versions.go

```diff
@@ -21,6 +21,7 @@ import (
 type Versions struct {
 	Product     product.Product
 	Constraints version.Constraints
+	Enterprise  EnterpriseOptions
```

Comment:
As mentioned we could turn this into a pointer, which could be equivalent to `Enterprise == true` but slightly simpler.